test/integration/test_serialization.py

```diff
@@ -200,6 +202,47 @@ def test_convert_complex(self):
         self.assertEqual(val[0], 0.2)
         self.assertEqual(val[1], 0.1)
 
+    def test_exception_message(self):
+        """Test executing job with Parameter in methadata."""
```

Comment:
Why did you add this test case to `test/integration` and not `test/unit`? The test does not appear to have a dependency on a remote system and can therefore be run as part of the unit tests that have no external dependencies, are more stable and considered for code coverage reporting.

Reply:
@kt474 yes these tests could be moved to `test/unit/test_serialization.py` (create a new file just for unit tests).